config/paths.go

```diff
@@ -39,7 +39,7 @@ var (
 	DefaultDebug = false
 	DefaultTrace = false
 
-	MaxReconnectAttempts             = uint(0) // 0 means to reconnect forever
+	MaxReconnectAttempts             = uint(10)
```

Comment:
Question: 
Why this change?

Reply:
From the commit message:
```
Abort connecting after 10 retries. Previously it would reconnect
forever with the same server URL, which will result in a deadlock if
the url (or the token) changed while re-connecting.
```

Reply:
Shame on me :see_no_evil: